shared/src/dto/user/user.dto.ts

```diff
@@ -78,6 +87,24 @@ export class DtoCreateUser {
   phoneNumber: string;
 }
 
+export class DtoCreateUserSignup {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsEmail()
+  email: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsString()
+  @Length(8, 64)
+  password: string | null;
```

Comment:
Although in TS we have specified that this field can be `null`, actually the validation doesn't allow it to be null